### PR TITLE
[15.0][IMP] hr_holidays_public: use api.model_create_multi

### PR DESCRIPTION
**Pull request by gurneyalex**
decorate hr.holidays.public.line::create with api.model_create_multi

### CODE REVIEW

File: hr_holidays_public/models/hr_holidays_public.py [hunk 1]
```diff
@@ -213,11 +213,13 @@ def _update_calendar_event(self):
             if rec.meeting_id:
                 rec.meeting_id.write(rec._prepare_holidays_meeting_values())
 
-    @api.model
+    @api.model_create_multi
+    @api.returns('self', lambda value: value.id)
```
Comment (pedrobaeza) [inline hr_holidays_public/models/hr_holidays_public.py]: I don't think you need to indicate this return decorator

File: hr_holidays_public/models/hr_holidays_public.py [hunk 2]
```diff
@@ -213,11 +213,13 @@ def _update_calendar_event(self):
...
     def create(self, values):
```
Comment (pedrobaeza) [inline hr_holidays_public/models/hr_holidays_public.py]: Follow super argument name:
```suggestion
    def create(self, vals_list):
```

File: hr_holidays_public/models/hr_holidays_public.py [hunk 3]
```diff
@@ -213,11 +213,13 @@ def _update_calendar_event(self):
...
         res = super().create(values)
-        res.meeting_id = self.env["calendar.event"].create(
-            res._prepare_holidays_meeting_values()
+        for record in res:
+            record.meeting_id = self.env["calendar.event"].create(
+            record._prepare_holidays_meeting_values()
```
Comment (pedrobaeza) [inline hr_holidays_public/models/hr_holidays_public.py]: Bad indentation

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: hr_holidays_public/models/hr_holidays_public.py (hunk 1 above)
Comment (gurneyalex) [reply hr_holidays_public/models/hr_holidays_public.py]: indeed... leftover from old times :)

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
We can fast-track this easy one:

/ocabot merge patch

**Pull request merged**